src/sagemaker/image_uri_config/tensorflow.json

```diff
@@ -2629,6 +2693,72 @@
                     "us-west-2": "763104351884"
                 },
                 "repository": "tensorflow-training"
+            },
+            "2.7.1": {
```

Comment:
Should this be `2.7.0` ?

Reply:
No, it's `2.7.1`. SageMaker DLCs for TF `2.7` were released after the first patch release for TF `2.7` was published, which meant that the "first" version of TF `2.7` SM DLCs was TF `2.7.1`, rather than `2.7.0`.